Remove server (#4984)
cyberbotics/webots: docs/guide/session-server.md

RewriteRule ^/session$ "http://localhost:1999/session" [P,L]
RewriteRule ^/(\d*)/(.*)$ "http://localhost:$1/$2" [P,L]
```

2. Configure the session server:

>     - Edit this file: `~/webots-server/config/session/session.json` so that it contains the following (to be adapted to your local setup):

Reviewer: This file doesn't exist:
```suggestion
    - Create a file named `~/webots-server/config/session/session.json` with the following contents (to be adapted to your local setup):
```
